controllers/barbican_controller.go

```diff
@@ -282,6 +283,9 @@ func (r *BarbicanReconciler) reconcileDelete(ctx context.Context, instance *barb
 func (r *BarbicanReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&barbicanv1beta1.Barbican{}).
+		Owns(&corev1.ServiceAccount{}).
+		Owns(&rbacv1.Role{}).
+		Owns(&rbacv1.RoleBinding{}).
```

Comment:
Hi, [kajinamit](https://github.com/kajinamit).  I was wondering why you're using `Owns` here instead of `Watches`.  I am still learning the framework, and if IIUC the Owns doc says these objects would be generated by the barbican operator?

> Owns defines types of Objects being *generated* by the ControllerManagedBy

https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.15.0/pkg/builder#Builder.Owns

Reply:
barbican controller DOES create these resources via ReconcileRbac .

Reply:
I see, thanks.